test/itest_dlc.py

```diff
@@ -0,0 +1,780 @@
+import testlib
+
+import time, datetime
+import json
+
+import pprint
+
+import requests # pip3 install requests
+
+import codecs
+
+deb_mod = False
```

Comment:
Does this stand for "debug mode"?

Reply:
Yes. That would not print too much when it is not necessary.